### ISSUE TITLE
RVL-1384 - Unable to edit family registration details for households where task has been completed in previous plans

### ISSUE THREAD

**Issue by samkanga** [OpenSRP/opensrp-client-reveal#1209]
For plans where only household registration activity takes place, if the household has completed registration in a previous plan the structure appears grey as there's no outstanding tasks to be done and none were generated.

Expected behaviour: User is able to view and edit household details. Structure is not greyed out

There were two possible options:

1. Dynamic tasking - could set rules for a new task (verify family) to be created
2. Allow grey structures to be clicked and present family module (edit family)
- Would be fine if the colour could be changed, but as colour is based on task business status how will this work…. there should be no grey structures unless the structure is archived. THerefore structures with a status of ‘Inactve’ should be gret, default colour for a structure should be pink (to indicate a family has been registered) - the structure will always be 
- Grey - deactivated
- Yellow - not registered
- Pink - family registered
- Other - depending on task status (purple etc

Option 2 is preferred by CHAI
